ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -157,8 +157,10 @@ object LedgerApiTestTool {
       if (config.included.isEmpty) defaultCases
       else allCases.filter(matches(config.included))
 
+    val addedTests = allCases.filter(matches(config.additional))
+
     val (excludedTests, testsToRun) =
-      includedTests.partition(matches(config.excluded))
+      (includedTests ++ addedTests).partition(matches(config.excluded))
```

Comment:
The behavior might be confusing now. If `--included` is added together with `--additional`, all the `additionals` become practically `--included`s, which defies the intention of this flag. I would disallow using `--included` with `--additional` together. Thus, we would also avoid silent mistakes that would lead to big parts of the test suite disabled.

Reply:
3919128